### ISSUE TITLE
JAWS breaks keyboard navigation when holding down left or right arrow on mass slider

### ISSUE THREAD

**Issue by loganbraywork** [phetsims/gravity-force-lab#235]
**Test device**
Windows 10 Laptop
**Operating System**
Windows 10
**Browser**
Chrome
**Problem description**
For  https://github.com/phetsims/QA/issues/471
Similar to the JAWS known screenreader bug of crashing when min/max value on a slider is exceeded
If JAWS is open, the mass slider is selected, and an arrow key is held down to manipulate the mass, then the slider will automatically drag itself to the min/max value and JAWS will stop responding.
To be clear, if the arrow is held down at all, it will continue to act as if it is being held down even if it is not and other inputs are made.
**Steps to reproduce**
1. Open JAWS
2. Use keyboard navigation to gain control of either Mass 1 or Mass 2 slider
3. Hold left or right arrow keys for about a second
4. Attempt to use keyboard navigation to switch view or continue to manipulate slider
**Visuals**
![2020-01-28GFLa11yJawsglitch](https://user-images.githubusercontent.com/54685448/73308089-c1e76780-41dc-11ea-92fb-17636220d61a.gif)
<details><summary>Visual Details</summary>For clarity, at around 400 mass is where I stopped holding down. Additional inputs by keyboard navigation all failed to produce any results.</details>

<details><summary><b>Troubleshooting information:</summary></b>
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-rc.1/phet/gravity-force-lab_all_phet.html
Version: 2.2.0-rc.1 2020-01-28 03:08:20 UTC
Features missing: generatedcontent, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36
Language: en-US
Window: 1536x754
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4095
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by zepumph** [comment]
So I didn't witness this behavior as a bug, but rather general lag with JAWs. It seems like holding down a key produces many DOM events, and then jaws takes all of them in a list and slowly releases them to the sim one at a time. This isn't really breaking other keyboard nav, it's just that those events will occur after all slider events are processed. I don't know if there is any way to get around this behavior since it is all happening before we (the sim) gets any events. 

I'd like to double check with @jessegreenberg first to see if we have encountered anything like this before, and second if he has any other thoughts or ideas.

**Comment by zepumph** [comment]
After discussing with @jessegreenberg in today's dev a11y meeting, we added this bug as (21) to the quirk bug tracking google doc https://docs.google.com/document/d/1518zv6F0odExFsodShZxwTmWNQk3civuNwtQUR-rEBs/edit.

@loganbraywork please close if that sounds good to you.
